balances-adjustment/src/main/java/com/powsybl/balances_adjustment/balance_computation/BalanceComputationImpl.java

```diff
@@ -82,7 +91,7 @@ public CompletableFuture<BalanceComputationResult> run(Network network, String w
             });
 
             // Step 2: compute Load Flow
-            LoadFlowResult loadFlowResult = loadFlowRunner.run(network, workingVariantCopyId, computationManager, parameters.getLoadFlowParameters());
+            LoadFlowResult loadFlowResult = loadFlowRunner.run(network, workingVariantCopyId, computationManager, parameters.getLoadFlowParameters(), reporter);
```

Comment:
Can you create a subReporter for the load flow (reporter.createSubReporter) ? This way the load flow part will be explictly separated in the report as more generic balance computation reports could be added later. (see how it is done for example here : https://github.com/powsybl/powsybl-open-loadflow/blob/323d052eeab1026f769addfca7a151a49d51855b/src/main/java/com/powsybl/openloadflow/sa/AcSecurityAnalysis.java#L170)

Reply:
@obrix I've created a subReporter, could you please check if it's implemented properly?